src/Orchard.Cms.Web/Modules/Orchard.Lists/Drivers/ListPartDisplayDriver.cs

```diff
@@ -94,7 +94,7 @@ private IEnumerable<ContentTypeDefinition> GetContainedContentTypes(ListPart lis
         {
             var contentTypeDefinition = _contentDefinitionManager.GetTypeDefinition(listPart.ContentItem.ContentType);
             var contentTypePartDefinition = contentTypeDefinition.Parts.FirstOrDefault(x => String.Equals(x.PartDefinition.Name, "ListPart", StringComparison.Ordinal));
-            var contentTypes = contentTypePartDefinition.Settings.ToObject<ListPartSettings>().ContainedContentTypes;
+            var contentTypes = contentTypePartDefinition.Settings.ToObject<ListPartSettings>().ContainedContentTypes ?? Enumerable.Empty<string>();
```

Comment:
You checked for a `null` value which was due to a NRE. Fixed it in the first place instead.